frontend/src/features/permits/hooks/cart.ts

```diff
@@ -90,7 +95,7 @@ export const useFetchCartItemStatus = () => {
 
   const cartItemDetailQuery = useQuery({
     queryKey: [CART_ITEM, cartItemId],
-    queryFn: () => getApplicationByPermitId(cartItemId),
+    queryFn: () => getApplication(cartItemId),
```

Comment:
This is because we need to consider all the possible implications of adding a possibly nullable companyId param throughout the application, and how the query hooks that use them are called (eg. here and in the other application details pages).